dev-docs/source/PyCharm.rst

```diff
@@ -23,7 +23,7 @@ Setting up PyCharm on Windows
 
    .. code-block:: sh
 
-      <Mantid Source Directory>/external/src/ThirdParty/lib/python2.7/python.exe
+      <Mantid Source Directory>/external/src/ThirdParty/lib/python3/python.exe
```

Comment:
This isn't quite right for windows as the directory is specifically `python3.8`

Reply:
Don't we use pyhton3.7 on ubuntu at the moment, though?